3ds2/src/main/java/com/adyen/checkout/adyen3ds2/internal/data/api/SubmitFingerprintRepository.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2021 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 21/5/2021.
+ */
+
+package com.adyen.checkout.adyen3ds2.internal.data.api
+
+import com.adyen.checkout.adyen3ds2.internal.data.model.SubmitFingerprintRequest
+import com.adyen.checkout.adyen3ds2.internal.data.model.SubmitFingerprintResult
+import com.adyen.checkout.core.action.data.RedirectAction
+import com.adyen.checkout.core.action.data.Threeds2Action
+import com.adyen.checkout.core.common.AdyenLogLevel
+import com.adyen.checkout.core.common.internal.helper.adyenLog
+import com.adyen.checkout.core.common.internal.helper.runSuspendCatching
+import org.json.JSONObject
+
+internal class SubmitFingerprintRepository internal constructor(
```

Comment:
if the class is internal, can we safely remove the internal modifier for the constructor?
```suggestion
internal class SubmitFingerprintRepository constructor(
```